### DISCUSSION TITLE
Animated image

### DISCUSSION THREAD

**Discussion by rwmitchell** [RhetTbull/osxphotos#1176] [General]
What is the point of the animated image on the project home page?

Only one example is shown and it would be nice to carefully read what is being displayed instead of having it constantly clear and restart.

**Comment by RhetTbull** [comment]
It's intended to give prospective users a peak at what osxphotos can do (in this case, a simple export example). That's it. I just watched it three times and it plays slow enough that I can read all the text. If you want to learn more, scroll on to the rest of the README and the tutorial which is quite detailed.
